src/Application.php

```diff
@@ -103,15 +104,18 @@ class Application implements
      * @param mixed $configuration
      * @param ServiceManager $serviceManager
      */
-    public function __construct($configuration, ServiceManager $serviceManager)
-    {
+    public function __construct(
+        $configuration,
+        ServiceManager $serviceManager,
+        EventManagerInterface $events,
```

Comment:
This is a BC break but we are working to mvc 3.0 if it is a GOOD BC break this is a good time to merge it